Build/Build.cs

```diff
@@ -302,6 +302,48 @@ from framework in supportedFrameworks
             ReportTestOutcome(projects.Select(p => $"*{p.Name}*.trx").ToArray());
         });
 
+    Target TestingPlatformFrameworks => _ => _
+        .DependsOn(Compile)
+        .OnlyWhenDynamic(() => RunAllTargets || HasSourceChanges)
+        .Executes(() =>
+        {
+            Project[] projects =
+            [
+                Solution.TestFrameworks.TUnit_Specs
+            ];
+
+            var testCombinations =
+                from project in projects
+                let frameworks = project.GetTargetFrameworks()
+                from framework in frameworks
+                select new { project, framework };
+
+            DotNetTest(s => s
+                .SetConfiguration(Configuration.Debug)
+                .SetProcessEnvironmentVariable("DOTNET_CLI_UI_LANGUAGE", "en-US")
+                .EnableNoBuild()
+                .CombineWith(
+                    testCombinations,
+                    (settings, v) => settings
+                        .SetProjectFile(v.project)
+                        .SetFramework(v.framework)
+                        .SetProcessArgumentConfigurator(args => args
+                            .Add("--")
+                            .Add("--coverage")
+                            .Add("--report-trx")
+                            .Add($"--report-trx-filename {v.project.Name}_{v.framework}.trx")
+                            .Add($"--results-directory {TestResultsDirectory}")
+                        )
+                    )
+                );
+
+            ReportTestOutcome(projects.Select(p => $"*{p.Name}*.trx").ToArray());
```

Comment:
Since https://github.com/fluentassertions/fluentassertions/pull/2776 this line is outdated.

Reply:
Removed